apps/api/src/app/testing/product-feature.e2e.ts

```diff
@@ -0,0 +1,32 @@
+import { OrganizationRepository } from '@novu/dal';
+import { ApiServiceLevelEnum } from '@novu/shared';
+import { UserSession } from '@novu/testing';
+import { expect } from 'chai';
+
+describe.only('Product feature Test', async () => {
```

Comment:
@rifont should we remove the only flag here?

Reply:
This has been removed.